# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../ezmlm/log_h.patch
# Copyright (C) 2004 - 2005 The T2 SDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---
--- ./ezmlm-manage.c.orig	2005-03-16 22:32:54.000000000 -0400
+++ ./ezmlm-manage.c	2005-03-16 22:33:27.000000000 -0400
@@ -22,6 +22,7 @@
 #include "fmt.h"
 #include "subscribe.h"
 #include "cookie.h"
+#include "log.h"
 
 #define FATAL "ezmlm-manage: fatal: "
 void die_usage() { strerr_die1x(100,"ezmlm-manage: usage: ezmlm-manage dir"); }
--- ./ezmlm-return.c.orig	2005-03-16 22:33:51.000000000 -0400
+++ ./ezmlm-return.c	2005-03-16 22:34:00.000000000 -0400
@@ -17,6 +17,7 @@
 #include "cookie.h"
 #include "subscribe.h"
 #include "issub.h"
+#include "log.h"
 
 #define FATAL "ezmlm-return: fatal: "
 void die_usage() { strerr_die1x(100,"ezmlm-return: usage: ezmlm-return dir"); }
